### PR TITLE
stability-tester: add sysbench test case 

### PR DESCRIPTION
**Pull request by cwen0**

### CODE REVIEW

**Comment by cwen0** [comment]
@siddontang @nolouch @disksing PTAL

**Comment by siddontang** [comment]
No select and delete?

File: stability-tester/config/suite.go [hunk 1]
```diff
@@ -82,3 +82,21 @@ type MVCCSuiteConfig struct {
 	Concurrency int            `toml:"concurrency"`
 	Bank        BankCaseConfig `toml:"bank"`
 }
+
+// SpecialSuiteConfig is the configuration for all special test cases
+type SpecialSuiteConfig struct {
```
Comment (siddontang) [inline stability-tester/config/suite.go]: what does special mean? this is not a good name. IMO, all the cases are special. Here the cases seem not to use the shared concurrent goroutines, but to run themselves in one goroutine.

Comment (cwen0) [reply stability-tester/config/suite.go]: Replace SpecialSuite with SerialSuite?

Comment (siddontang) [reply stability-tester/config/suite.go]: ok, please add a comment what is the Suite suitable for.

File: stability-tester/config/config.go [hunk 2]
```diff
@@ -47,11 +47,12 @@ type Config struct {
...
-	Cluster   ClusterConfig   `toml:"cluster"`
-	Nemeses   NemesesConfig   `toml:"nemeses"`
-	Suite     SuiteConfig     `toml:"suite"`
-	MVCC      MVCCSuiteConfig `toml:"mvcc"`
-	Scheduler SchedulerConfig `toml:"scheduler"`
+	Cluster      ClusterConfig      `toml:"cluster"`
+	Nemeses      NemesesConfig      `toml:"nemeses"`
+	Suite        SuiteConfig        `toml:"suite"`
+	MVCC         MVCCSuiteConfig    `toml:"mvcc"`
+	SerialSuite SerialSuiteConfig `toml:"serial_suite"`
```
Comment (disksing) [inline stability-tester/config/config.go]: Seems not well formatted.

**Review by disksing — COMMENTED** [review]
the rest LGTM.

**Review by siddontang — APPROVED** [review]
LGTM

**Pull request merged**